docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,209 @@
+
+The new **PMM Agent down** alert template monitors your Node status to notify if the Agent is down. Before using this template:
+
+ - ensure that the PMM Agent is healthy and present on the Database Node.
+-  check that PMM Client is on version 2.38 version or later.
+
+![!image](../_images/Agent_down.png)
+
+## New Features
+
+- [PMM-2852](https://jira.percona.com/browse/PMM-2852) - [MySQL, PostgreSQL] QAN support for parsing query comments
+- [PMM-11961](https://jira.percona.com/browse/PMM-11961) - Starting with PMM 2.38.0, we have enhanced the **Inventory** page by improving the **Nodes** tab. With this change, you can get more context on your inventory objects, such as node, and more actionable information for exploring and resolving possible issues.
+- [PMM-11962](https://jira.percona.com/browse/PMM-11962) - Starting with PMM 2.38.0, you can filter the nodes and services on the **Inventory** page using the parameters such as ID, names, status, etc., instead of scrolling through an exhaustive list of nodes or services.
+- [PMM-12077](https://jira.percona.com/browse/PMM-12077) - Telemetry data about the base image
+- [PMM-12087](https://jira.percona.com/browse/PMM-12087) - Added tooltips to explain the meaning of the different services statuses in the Inventory view.   
+
+
+## Improvements
+
+- [PMM-9367](https://jira.percona.com/browse/PMM-9367) - MongoDB Backup: Sharded Cluster Backup
+- [PMM-11250](https://jira.percona.com/browse/PMM-11250) - Allow user to specify target directory for backup artifact
+- [PMM-12038](https://jira.percona.com/browse/PMM-12038) - Extended support for MongoDB backups to cover all authentication mechanisms available for MongoDB (including LDAP, x509 certs).
+- [PMM-12114](https://jira.percona.com/browse/PMM-12114) - Grafana has been upgraded to 9.2.18 due to CVE.
+- [PMM-12136](https://jira.percona.com/browse/PMM-12136) - We have migrated our base operating system on which we build our base image from Enterprise Linux 7 (CentOS 7) to Oracle Linux 9 to fix the CVE's.
+
+## Bugs Fixed
+
+- [PMM-11578](https://jira.percona.com/browse/PMM-11578) - The **Last notified** column  under **Alerting > Fired Alerts** has been renamed to **Last triggered**. This better illustrates the fact that triggered alerts may not always generate a notification. This can happen when the contact points are not correctly configured.
+In addition, the **Suppressed** status in the **State** column has been renamed to **Silenced** to keep consistency with how this status is displayed on other **Alerting** pages.
+
+- [PMM-11645](https://jira.percona.com/browse/PMM-11645) - Fixed mysql service detection when this is not loaded in systemd.
+- [PMM-9541](https://jira.percona.com/browse/PMM-9541) - PostgreSQL exporter not working with --socket
+- [PMM-9544](https://jira.percona.com/browse/PMM-9544) - Alerting doesn't seem to be able to monitor for "Host Down"
```

Comment:
This moved to Improvements  :  New PMM Agent Down alert template

Reply:
@catalinaadam Can you please check this?